pkg/cloudprovider/vsphere/route/routeprovider_test.go

```diff
@@ -208,6 +281,24 @@ func TestCreateRoute(t *testing.T) {
 	mockBroker.EXPECT().ListRealizedEntities(routeID).Times(0)
 }
 
+func TestGenerateStaticRoute(t *testing.T) {
```

Comment:
Can we also test IPv6 cidrs / node IPs here?

Reply:
Sure